cli/snyk.js

```diff
@@ -67,6 +68,17 @@ module.exports = class Snyk {
             })
         ).issues;
     }
+
+    async get(url) {
+        return (
+            await request({
+                method: 'get',
+                url,
+                headers: this._headers,
+                json: true,
+            })
+        );
+    }
```

Comment:
Note to reviewers:

This new `get` function is used to crawl links returned by the API. See the `links` property in the "[List all Aggregated Issues](https://snyk.docs.apiary.io/#reference/projects/aggregated-project-issues/list-all-aggregated-issues)" or "[List all project issue paths](https://snyk.docs.apiary.io/#reference/projects/project-issue-paths/list-all-project-issue-paths)" docs.

Reply:
Maybe we can make the name more descriptive, like `get/loadLink` or something like that?

Reply:
My idea was to name it after the `GET` http method, but I can see that it's a little generic out of context. I like `getLink` 👍